politeiawww/api/v1/api.md

```diff
@@ -1143,378 +799,44 @@ Reply:
 
 ### Error codes
```

Comment:
The links to individual error codes at the top and throughout the doc seem to be broken.